Tutorials/Python_Developer_Tutorials/foundation/Commanding-Local/Proxy-Local-Commands-Tutorial_.py

```diff
@@ -1,87 +1,110 @@
+import os
+import sys
+import time
+import ConfigParser
+
+sys.path.append(os.path.join(os.path.dirname(__file__),
+                             "../../../../Faraday_Proxy_Tools"))  # Append path to common tutorial FaradayIO module
 
-#Imports - Faraday Specific
+# Imports - Faraday Specific
 from FaradayIO import faradaybasicproxyio
 from FaradayIO import faradaycommands
 from FaradayIO import gpioallocations
 
-import time
-
-#Definitions
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("configuration.ini")
+config.read(filename)
 
+# Definitions
 
-#Variables
-local_device_callsign = 'REPLACEME'  # Should match the connected Faraday unit as assigned in Proxy configuration
-local_device_node_id = REPLACEME  # Should match the connected Faraday unit as assigned in Proxy configuration
+# Variables
+local_device_callsign = config.get("DEVICES",
+                                   "UNIT0CALL")  # Should match the connected Faraday unit as assigned in Proxy configuration
+local_device_node_id = config.get("DEVICES",
+                                  "UNIT0ID")  # Should match the connected Faraday unit as assigned in Proxy configuration
+local_device_callsign = str(local_device_callsign).upper()
 
-#Start the proxy server after configuring the configuration file correctly
-#Setup a Faraday IO object
-faraday_1 = faradaybasicproxyio.proxyio() #default proxy port
+# Start the proxy server after configuring the configuration file correctly
+# Setup a Faraday IO object
+faraday_1 = faradaybasicproxyio.proxyio()  # default proxy port
 faraday_cmd = faradaycommands.faraday_commands()
 
-
 ##############
 ## TOGGLE GPIO
 ##############
-#Turn LED 1 ON LOCAL
+# Turn LED 1 ON LOCAL
 print "Turning ON the Green LED (LED #1)"
 command = faraday_cmd.CommandLocalGPIOLED1On()
 faraday_1.POST(local_device_callsign, local_device_node_id, faraday_1.CMD_UART_PORT, command)
 time.sleep(1)
 
-#Turn LED 1 OFF LOCAL
+# Turn LED 1 OFF LOCAL
 print "Turning OFF the Green LED (LED #1)"
 command = faraday_cmd.CommandLocalGPIOLED1Off()
 faraday_1.POST(local_device_callsign, local_device_node_id, faraday_1.CMD_UART_PORT, command)
 
-#Turn LED 2 ON LOCAL
+# Turn LED 2 ON LOCAL
 print "Turning ON the Red LED (LED #2)"
 command = faraday_cmd.CommandLocalGPIOLED2On()
 faraday_1.POST(local_device_callsign, local_device_node_id, faraday_1.CMD_UART_PORT, command)
 time.sleep(1)
 
-#Turn LED 2 OFF LOCAL
+# Turn LED 2 OFF LOCAL
 print "Turning Off the Red LED (LED #2)"
-command = faraday_cmd.CommandLocalGPIO(0, 0, 0, gpioallocations.LED_2, 0, 0) #This examples how the non predefined LED GPIO commanding is created. Multiple GPIO's can be toggled at once using ||'s
+command = faraday_cmd.CommandLocalGPIO(0, 0, 0, gpioallocations.LED_2, 0, 0)  # This examples how the non predefined LED GPIO commanding is created.
```

Comment:
wait why not use `CommandLocalGPIOLED2Off()?` like you used for LED1 since you used a similar command for LED1? Based on inline comment that's the goal?